app/src/main/java/org/glucosio/android/tools/GlucosioConverter.java

```diff
@@ -34,9 +35,8 @@ public static double round(double value, int places) {
         return bd.doubleValue();
     }
 
-    public static int glucoseToMgDl(double mmolL) {
-        double converted = mmolL * 18;
-        return (int) converted;
+    public static double glucoseToMgDl(double mmolL) {
+        return mmolL * 18;
```

Comment:
create a constant so it make's sense? I have no idea what is 18 atm

Reply:
Added constant for 18